test/external/clone.js

```diff
@@ -7,22 +7,23 @@
  */
 
 import { cpus } from 'node:os'
-import { join, dirname } from 'node:path'
 import { readFileSync, mkdirSync, access, R_OK, W_OK, writeFileSync } from 'node:fs'
-import { fileURLToPath } from 'node:url'
 import crossSpawn from 'cross-spawn'
 import minimist from 'minimist'
 import parallelLimit from 'run-parallel-limit'
 import test from 'tape'
 
-const __dirname = dirname(fileURLToPath(import.meta.url))
-const json = readFileSync(join(__dirname, 'test.json'), 'utf8')
+const getAbsolutePath = (path, pwd = import.meta.url) =>
+  new URL(path, pwd).toString().slice(7)
+
+const testJsonPath = getAbsolutePath('test.json')
```

Comment:
(1/3)

I think it would be better to keep the variables as `URL` instead of strings as long as possible, instead of the other way around. That is, this variable can be an `URL` and be passed as that to `readFileSync` and `writeFileSync`.

```suggestion
const testJsonPath = new URL('test.json', import.meta.url)
```